README.md

```diff
@@ -92,6 +92,22 @@ If you use MySQL 8, please read [this link](https://www.php.net/manual/en/mysqli
 
 7. Follow the [Setup Guide in the LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) to complete your post-installation setup and configuration, and for more documentation.
 
+### Upgrade Step
```

Comment:
I think this should be a one line mention of this step/process/script -- and point to the rest of this content under the markdown docs/ (what Rida calls the "new wiki)